apps/test-bundles/webpack.config.js

```diff
@@ -1,8 +1,31 @@
 // @ts-check
-const { createWebpackConfig, buildEntries, buildEntry } = require('./webpackUtils');
+const {
+  buildEntries,
+  buildEntry,
+  createWebpackConfig,
+  createFluentFixtures,
+  createFluentNorthstarFixtures,
+  createEntry,
+} = require('./webpackUtils');
+
+const package = process.env.PACKAGE;
```

Comment:
I'd probably prefer a command line arg rather than an environment variable, but that's not essential. (If you do a command line arg, for a simple case like this you can just find it from `process.argv` directly rather than using yargs or something.)

In either case, it would be good to verify that the arg is specified and `process.exit(1)` if not.

Reply:
@ecraig12345 I couldn't access the command line argument in the webpack config neither by process nor with yargv. The only place I managed to access it was in just.config.ts but didn't find a way to pass it from there to webpack. If you know how, I am happy to implement that, also think it would be cleaner.

And implemented the process.exit to the code 👍

Reply:
Ah right, I forgot about that. Environment variable is fine then.